packages/editor/src/editor/browser/data-class/DataClassDialog.tsx

```diff
@@ -44,18 +44,8 @@ export const DataClassDialog = ({
     </DialogContent>
   </Dialog>
 );
-export const getSelectedElementId = (selectedElementId: string | undefined) => {
-  if (
-    selectedElementId &&
-    (selectedElementId.startsWith('Layout') || selectedElementId.startsWith('Panel') || selectedElementId.startsWith('Fieldset'))
-  ) {
```

Comment:
this is not safe, as there is no guarantee that this id's starts with 'Layout', 'Panel' or 'Fieldset'.